prov/rxm/src/rxm_ep.c

```diff
@@ -113,6 +113,11 @@ static int rxm_buf_reg(struct ofi_bufpool_region *region)
 	struct rxm_domain *rxm_domain;
 	int ret;
 
+	ret = ofi_hmem_host_register(region->mem_region,
```

Comment:
I think we need to check if FI_HMEM has been enabled by the application prior to calling this.

Reply:
Can do.